### PR TITLE
Two unrelated UX tweaks

### PR DESCRIPTION
**Pull request by NiLuJe**
Re #6572 & #7497

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7505)

### CODE REVIEW

**Comment by NiLuJe** [comment]
Sidebar: the highlight stuff is weird. The highlight type is baked into the HL themselves, and the lighten factor is only read at loading time (e.g., requires a restart).

**Comment by poire-z** [comment]
> and the lighten factor is only read at loading time (e.g., requires a restart).
> Sidebar: should this (and the other page_gap_color hidden setting) be read at :init instead?

The other callbacks (for highlight style/disable) update directly `self.view.highlight.saved_drawer = style`, so in your callback, you could do the same: 
`self.view.highlight.lighten_factor = spin.value`
and it would then be used without a restart (dunno if it would update the page, or if you'd need a refresh/repaint to have it applied live).

File: frontend/apps/reader/modules/readerhighlight.lua [hunk 1]
```diff
@@ -323,14 +323,45 @@ function ReaderHighlight:genHighlightDrawerMenu()
...
                 self:toggleDefault()
             end,
             separator = true,
         },
         get_highlight_style("lighten"),
         get_highlight_style("underscore"),
         get_highlight_style("invert"),
+        {
+            text_func = function()
+                return T(_("Lightening factor: %1"), G_reader_settings:readSetting("highlight_lighten_factor", 0.2))
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readerhighlight.lua:336]: confusing name

Comment (NiLuJe) [reply frontend/apps/reader/modules/readerhighlight.lua:336]: True, but haven't really found anything better :/.

Comment (NiLuJe) [reply frontend/apps/reader/modules/readerhighlight.lua:336]: `Highlight contrast`, maybe?

Comment (NiLuJe) [reply frontend/apps/reader/modules/readerhighlight.lua:336]: (It's grayed out in other modes, so it doesn't really need to match/contain "lighten" to make it clear that applies to the ligthen mode).

Comment (poire-z) [reply frontend/apps/reader/modules/readerhighlight.lua:336]: Highlight opacity? (as "the higher the darker")

Comment (NiLuJe) [reply frontend/apps/reader/modules/readerhighlight.lua:336]: Works for me, yeah ;).

Comment (Frenzie) [reply frontend/apps/reader/modules/readerhighlight.lua]: Yes, I suppose that'll do.

**Pull request merged**